packaging/debian-unstable/snapd.postrm

```diff
@@ -0,0 +1,87 @@
+#!/bin/sh
```

Comment:
Inlining the script would be dumb, but refactoring the Debian packaging scripts to leverage the snap-mgmt script would be something to strive toward.